packages/web3-eth-contract/src/contract.ts

```diff
@@ -1004,7 +826,8 @@ export class Contract<Abi extends ContractAbi>
 			return validatorUtils.transformJsonDataToAbiFormat(abi.inputs ?? [], params);
 		} catch (error) {
 			throw new Web3ContractError(
-				`Invalid parameters for method ${abi.name}: ${(error as Error).message}`,
+				// todo undo formatting, vscode bug
+				`Invalid parameters for method ${abi.name}: ${(error as Error).message}\``,
```

Comment:
We should remove this right? And just add it back if we have to work on this file before VSCode fixes the issue?

```suggestion
				`Invalid parameters for method ${abi.name}: ${(error as Error).message}`,
```